src/lib/transaction_logic/test/transaction_logic.ml

```diff
@@ -239,3 +239,41 @@ let%test_module "Test transaction logic." =
                  Transaction_status.equal txn.command.status Applied ) )
             (run_zkapp_cmd ~fee_payer ~fee ~accounts txns) )
   end )
+
+(* This module tests Inputs.Stack *)
+let%test_module "Test stack module" =
+  ( module struct
+    module Stack = Transaction_logic.For_tests.Stack (Int)
+
+    let%test_unit "Ensure pop works on non-empty list." =
+    Quickcheck.test ~trials 
```

Comment:
Should this not be indented?

Reply:
I believe you're correct. The formatting looks off. I'll fix this now.

Reply:
Fixed with commit 02b8b867c660c114871de79bda2dc2344e3e7d49.